input/symptoms/watchdogTimeout.ts

```diff
@@ -17,8 +18,12 @@ export default {
 			'https://jel.ly.fish/pattern-systemd-watchdog-kills-balenaengine-extended-heavy-load-6min--33f2d65',
 		watchdogTimeout: {
 			description: 'Engine killed by the watchdog',
-			$$formula:
-				"/balena.service: Watchdog timeout/.test(contract['journalctl --no-pager --no-hostname -n 1000 -at balenad'].stdout)",
+			$$formula: toFormula((contract) =>
```

Comment:
It would also be nice to be able to pass the function directly to $$formula and let the parser handle the `toFormula` bit